modules/extension/wmts/src/main/java/org/geotools/ows/wmts/request/AbstractGetTileRequest.java

```diff
@@ -62,9 +61,11 @@
 
     static WMTSTileFactory factory = new WMTSTileFactory();
 
-    public static final String LAYER = "Layer";
+    public static final String LAYER = "layer";
```

Comment:
Keys in OWS GET requests are supposed to be case insensitive, so the change should not be necessary (unless you're trying to deal with a broken implementation). Also, keys before the change were all following the same case, now we have a mix of all lowercase and camelcase.

Reply:
It was a test that failed, but I've changed the assumption of that one instead.